### ISSUE TITLE
Theme: Users with expired passwords see blank page

### ISSUE THREAD

**Issue by Arcanum417** [qooob/authentic-theme#554]
Webmin with expired passwords (Force change at next login) cannot change them at next login. They can only see blank page. I have "Prompt users with expired passwords to enter a new one" selected in webmin configuration.

Blank page

```
<!DOCTYPE html>
<html data-background-style="lightGrey">
<head>
<title data-initial=""></title>
<meta charset="UTF-8">
<link rel="shortcut icon" href="/images/favicon-webmin.ico">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link href="/unauthenticated/css/bootstrap.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/bootstrap.tagsinput.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/datepicker.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/fontawesome-animation.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/codemirror.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/jquery.jspanel.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/jquery.datatables.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/nprogress.min.css?1810" rel="stylesheet" type="text/css">
<link href="/unauthenticated/css/authentic.min.css?1810" rel="stylesheet" type="text/css">
<script src="/unauthenticated/js/settings.js?1475168263" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery-ui.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.jspanel.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.scrollintoview.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/momentjs.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/bootbox.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.purl.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/bootstrap.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/bootstrap.tagsinput.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/datepicker.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/fileinput.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/codemirror.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.datatables.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.datatables.plugins.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/jquery.easypiechart.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/clipboard.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/contextmenu.min.js?1810" type="text/javascript"></script>
<script src="/unauthenticated/js/init.min.js?1810" type="text/javascript"></script>
</head>
<body data-debug="prod" data-usermin="0" data-shell="0" data-theme="blue" data-level="0" data-user="" data-dashboard="1" data-language="en" data-charset="UTF-8" data-webprefix="" data-current-product="webmin" data-style="-webkit-filter: grayscale(0)  saturate(1) hue-rotate(0deg); filter: grayscale(0)  saturate(1) hue-rotate(0deg);">
<div class="container-fluid col-lg-10 col-lg-offset-1">
<div class="panel panel-default">
<div class="panel-heading">
<table class='header' width=100%><tr>
<td id="headln2l" width="25%" align="left" class="invisible"></td>
<td id='headln2c' align=center width=50%><font size=+2></font></td>
<td id='headln2r' width=25% align=right></td></tr></table>
</div>
<div class="panel-body">
<center>
<h3 class='ui_subheading'>You must select a new password to replace your temporary login.</h3>

<form class="ui_form" role="form" action="/password_change.cgi" method="post" >
<input class='ui_hidden' type='hidden' name="user" id="user" value="vaclav">
<input class='ui_hidden' type='hidden' name="pam" id="pam" value="1">
<input class='ui_hidden' type='hidden' name="expired" id="expired" value="2">
<div class='table-responsive'><table class='table table-striped table-condensed table-subtable' >
<thead><tr><th class='table-title'><b>Select New Password</b></th></tr></thead>
<tbody> <td><table class='sub_table_container' width=100%>
<tr>
<td class='col_label'><b>Username</b></td>
<td colspan="1" class="col_value">vaclav</td></tr>
<tr>
<td class='col_label'><b>Current password</b></td>
<td colspan="1" class="col_value"><input style="display: inline; width: auto; height: 28px; padding-top: 0; padding-bottom: 0; vertical-align:middle" class="form-control ui_password" type="password" name="old" value="" size="20" >
</td></tr>
<tr>
<td class='col_label'><b>New password</b></td>
<td colspan="1" class="col_value"><input style="display: inline; width: auto; height: 28px; padding-top: 0; padding-bottom: 0; vertical-align:middle" class="form-control ui_password" type="password" name="new1" value="" size="20" >
</td></tr>
<tr>
<td class='col_label'><b>New password again</b></td>
<td colspan="1" class="col_value"><input style="display: inline; width: auto; height: 28px; padding-top: 0; padding-bottom: 0; vertical-align:middle" class="form-control ui_password" type="password" name="new2" value="" size="20" >
</td></tr>
</table></td></tbody></table></div>
<table class='ui_form_end_buttons' ><tr><td>
<div class="btn-group end_submits"><button class="btn btn-default ui_submit ui_form_end_submit" type="button">&nbsp;<span data-entry="password_ok">Change&nbsp;</span></button>
<input class="hidden" type="submit" >
</div></td></tr></table>
</form>
<script type='text/javascript'>
var opts = document.getElementsByClassName('ui_opt_textbox');
for(var i=0; i<opts.length; i++) {
  opts[i].disabled = document.getElementsByName(opts[i].name+'_def')[0].checked;
}
</script>
</center>

</div>
<script src="/unauthenticated/js/postinit.min.js?1810" type="text/javascript"></script><script>___authentic_theme_footer___ = 1;</script>
<script src="/unauthenticated/js/parent.min.js?1810" type="text/javascript"></script>
</body>
</html>
```
Provide the following information:
* What is your server-side OS and its version? 
Distributor ID: Ubuntu
Description:    Ubuntu 15.10
Release:        15.10
Codename:       wily

* What is your client-side OS, browser and its versions?
Chrome 53.0.2785.116 m (64-bit)
* What are the steps to reproduce the issue
^^ up there

**Comment by Arcanum417** [comment]
It works as intended

**Comment by qooob** [comment]
I'll take a look, thanks.

**Comment by Arcanum417** [comment]
Hello, whats the status on this ? Its kinda annoying.

**Comment by qooob** [comment]
Working on crazy CSF fix. This is next in the line.

It will be released, by my premonitions, in the end of October. Please be patient. It's lots to do.

**Comment by Arcanum417** [comment]
Just checking, thx for the effort.

**Comment by qooob** [comment]
It will be fixed in 18.20.

Thanks for reporting!!

**Comment by qooob** [comment]
Version 18.20 is out, please upgrade.
